spec/bullet_spec.rb

```diff
@@ -38,4 +38,22 @@
       end
     end
   end
+
+  describe '#start?' do
+    context 'when bullet is disabled' do
+      around(:each) do |example|
+        begin
+          old_enabled = Bullet.enable?
+          Bullet.enable = false
+          example.run
+        ensure
+          Bullet.enable = old_enabled
+        end
+      end
+
+      it 'should not be started' do
+        expect(Bullet.start?).to be(false)
+      end
+    end
+  end
```

Comment:
I mean you can add the following code to spec_helper

```
config.before :each do
  Bullet.enable = true
end
```

then in you test you can say 

```
Bullet.enable = false
expect(Bullet.start?).to be(false)
```

a bit cleaner